server/game/gamesteps/dominancephase.js

```diff
@@ -35,6 +35,8 @@ class DominancePhase extends Phase {
         }
 
         if(dominanceWinner) {
+            //save the winner of dominance on the game object in order to use this information in determining the winner of the game after the time limit has expired
+            this.game.winnerOfDominanceInLastRound = dominanceWinner;
```

Comment:
Good call on placing this outside the `if`. Rules state you can win dominance even if you can't gain power. 👍